woocommerce/woocommerce
# CI: reduce running time for PHP unit testing jobs

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Reduce execution times for PHPUnit-related jobs (5 currently) by skipping the build step (mostly about building assets). Build times vary, but on average, they're around 2.5 minutes (?).

Full scope:
- PHPUnit and API tests: skipping building assets (no interactions with UI)
- Linting: skipping building assets (not needed to perform linting)

### How to test the changes in this Pull Request:

See [here](https://github.com/woocommerce/woocommerce/actions/runs/9793406791/job/27041437354?pr=49135) and note `Build project` is skipped in `PHP *` jobs, which are green.
